crates/sui-json-rpc/src/api.rs

```diff
@@ -496,10 +496,22 @@ pub trait EventReadApi {
     ) -> RpcResult<Vec<SuiEventEnvelope>>;
 }
 
-#[open_rpc(namespace = "sui", tag = "Quorum Driver APIs to execute transactions.")]
+#[open_rpc(namespace = "sui", tag = "APIs to execute transactions.")]
 #[rpc(server, client, namespace = "sui")]
-pub trait QuorumDriverApi {
-    /// Execute the transaction and wait for results if desired
+pub trait TransactionExecutionApi {
+    /// Execute the transaction and wait for results if desired.
+    /// Request types:
+    /// 1. ImmediateReturn: immediate return to client without waiting for any results
+    ///     Note the transaction may fail without being noticed by client in this mode.
+    ///     After getting the response, client use poll the node to check transaction's result
```

Comment:
After getting the response, the client polls the node to checks the result of the transaction

Reply:
I added a commit with this change